aces/Loris
# [Test] unittests for Database.class.inc

## Brief summary of changes

Unit tests for the Database library under `php/libraries`. This PR will be updated as more unit test coverage is added. Comments were added to pre-existing and new unit tests.

setUp and tearDown functions were added as well.

As of 06/16: All **public** methods are covered except for `connect` and `singleton`. 

#### Testing instructions

1. Run `npm run tests:unit -- --filter Database` or check out the Travis output.
